sphinx/tutorial/cylc/runtime/introduction.rst

```diff
@@ -340,6 +341,8 @@ Suite Files
    ``share/``
       The :term:`share directory` is a place where :term:`tasks <task>` can
       write files which are intended to be shared within that cycle.
+   ``work/``
+      A directory hierarchy containing tasks :term:`work directory`.
```

Comment:
The 'term' / glossary link just sits weirdly at the end of the sentence here. I suggest rephrasing as in the the ``share/`` description item just above to incorporate the term naturally.

Reply:
Apologies, I misunderstood what was meant here looking at your feedback change. The apostrophe added does make it more obvious :)